tasks/scripting/src/test/java/org/infinispan/scripting/ScriptMetadataTest.java

```diff
@@ -84,29 +86,36 @@ public void testDataTypeOther() throws Exception {
       assertEquals(StandardCharsets.US_ASCII, metadata.dataType().getCharset());
    }
 
-   public void testArrayValues() throws Exception {
+   public void testArrayValues() {
       ScriptMetadata metadata = ScriptMetadataParser.parse("test.scala", "// name=test,language=javascript,parameters=[a,b,c]");
       assertEquals("test", metadata.name());
       assertEquals("javascript", metadata.language().get());
-      assertTrue(metadata.parameters().containsAll(Arrays.asList("a", "b", "c")));
+      assertThat(metadata.parameters()).containsExactly("a", "b", "c");
+   }
+
+   public void testProperties() {
+      ScriptMetadata metadata = ScriptMetadataParser.parse("test.scala", "// name=test,language=javascript,properties={k1:v1,k2:v2}");
```

Comment:
The new log message has an example where the properties are wrapped in quotes `{"k1": "v1", "k2": "v2"}`, isn't it needed? That's the reason in my other comment as well.